app/main/auto-updater/config.js

```diff
@@ -0,0 +1,8 @@
+const config = {
+  baseFeedUrl: `https://hazel-server-pxufsrwofl.now.sh`,
```

Comment:
is this a stable url? should we pay for the ability to run it on say update.appium.io?

Reply:
@jlipps It's hosted by https://zeit.co/now... not sure how stable that is. I ran this server by following the instructions on zeit/hazel git page.

Reply:
The URL itself is permanent and I'm pretty sure the server should be up-and-running permanently; if it ever were taken down, I think we would get notice first.